lighthouse-cli/test/smokehouse/test-definitions/redirects/expectations.js

```diff
@@ -5,24 +5,22 @@
  */
 'use strict';
 
-const cacheBuster = Number(new Date());
-
 /**
  * @type {Array<Smokehouse.ExpectedRunnerResult>}
  * Expected Lighthouse audit values for redirects tests
  */
 const expectations = [
   {
+    // Single server-side redirect (2s)
     lhr: {
-      requestedUrl: `http://localhost:10200/online-only.html?delay=500&redirect=%2Foffline-only.html%3Fcb=${cacheBuster}%26delay=500%26redirect%3D%2Fredirects-final.html`,
+      requestedUrl: `http://localhost:10200/online-only.html?delay=2000&redirect=%2Fredirects-final.html`,
       finalUrl: 'http://localhost:10200/redirects-final.html',
       audits: {
         'redirects': {
-          score: '<1',
-          numericValue: '>=500',
```

Comment:
this numericValue was misleading  because it has nothing to do with `delay` it's lantern-based, future followup resolves this issue in `redirects` audit, but just reorganizing the checks here first.